src/access.ts

```diff
@@ -0,0 +1,18 @@
+/** @module access */
+
+/** @constant {string} */
+const UNKNOWN = 'unknown';
+
+/** @constant {string} */
+const USER = 'user';
+
+/** @constant {string} */
+const ORGANIZATION = 'organization';
+
+/** @constant {string} */
+const PUBLIC = 'public';
+
+/** @constant {string[]} */
+const access = [UNKNOWN, USER, ORGANIZATION, PUBLIC];
+
+export { UNKNOWN, USER, ORGANIZATION, PUBLIC, access };
```

Comment:
I haven't applied this pattern to the categories yet, but Typescript has support for string enumerations. I think we should use them here too. In this case it'd look like:
```suggestion
export enum Access {
  Unknown = 'unknown',
  User = 'user',
  Organization = 'organization',
  Public = 'public',
}
```
Users can then import the `Access` enum and reference its properties like this:
```typescript
import { Access } from './access';

function doTheThing(access: Access): void {
  console.log(access);
}
doTheThing(Access.User); // Logs 'user' to console.
```
and TS then offers more explicit typechecking as opposed to merely checking that it's a string.